docs/Troubleshooting.md

```diff
@@ -1,26 +1,34 @@
 # Troubleshooting
-**Run `brew update` (twice) and `brew doctor` *before* creating an issue!**
```

Comment:
Would rather leave this at the top as people often don't read more than the first line here, sadly.